Src/FluentAssertions/Execution/ContinuedAssertionScope.cs

```diff
@@ -21,81 +21,32 @@ internal ContinuedAssertionScope(AssertionScope predecessor, bool continueAssert
     }
 
     /// <inheritdoc/>
-    public GivenSelector<T> Given<T>(Func<T> selector)
-    {
-        if (continueAsserting)
-        {
-            return predecessor.Given(selector);
-        }
-
-        return new GivenSelector<T>(() => default, predecessor, continueAsserting: false);
-    }
+    public GivenSelector<T> Given<T>(Func<T> selector) =>
+        continueAsserting ? predecessor.Given(selector) : new GivenSelector<T>(() => default, predecessor, continueAsserting: false);
```

Comment:
This I most definitely _do not_ agree with.

Reply:
I agree, when either of the two expressions get just a bit "complex", I would prefer not to use a ternary operator, and I imagine it gets too vague for an analyzer to estimate when an expression is complex.

I'm fine/indifferent about the changes `Comparands.cs` and `MethodInfoExtensions.cs`.
Let's revert the changes in `Tracer.cs` and `ContinuedAssertionScope.cs`

.. and perhaps disable the triggered Quodana rule.